hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/ContainerEndpoint.java

```diff
@@ -104,6 +108,8 @@ public class ContainerEndpoint {
   private static final Logger LOG =
       LoggerFactory.getLogger(ContainerEndpoint.class);
   private BucketLayout layout = BucketLayout.DEFAULT;
+  private static final String SCM_FILTER = "SCM";
```

Comment:
This is not enum

Reply:
Sorry I mistook the suggestion as static variables, I have implemented an Enum for the filter.